test/test_scale_dependent_statistics.py

```diff
@@ -53,7 +54,27 @@ def test_scalar_input():
         iter(s3)
 
 
-def test_nonuniform():
+def test_uniform_synthetic():
+    t = fourier_synthesis((1024,), (7,), 0.8, rms_slope=0.1, periodic=False).detrend()
+
+    L, u = t.bandwidth()
+
+    r, A = t.autocorrelation_from_profile(resampling_method=None)
+    r = r[1:]
+    A = A[1:]
+    assert abs(r.min()/L - 1) < 0.1  # Within 10% of bandwidth
+    assert abs(r.max()/u - 1) < 0.1
+
+    f = scipy.interpolate.interp1d(r, np.sqrt(2 * A) / r)
+    r, s = t.scale_dependent_statistical_property(lambda x: np.mean(x * x), n=1)
+
+    assert abs(r.min()/L - 1) < 0.1  # With 10% of bandwidth
+    assert abs(r.max()/u - 1) < 0.1
+
+    np.testing.assert_allclose(np.sqrt(s), f(r), atol=0.01)
+
+
+def test_nonuniform_synthetic(plot=True):
```

Comment:
should we deactivat the plot, or is that overriden by a fixture ?

Reply:
There was no plotting, I've removed the argument